src/api/veilarbperson.ts

```diff
@@ -1,6 +1,9 @@
 import { AxiosPromise } from 'axios';
 import { axiosInstance } from './utils';
 import { ArbeidssokerPeriode } from '@navikt/arbeidssokerregisteret-utils';
+import { BEHANDLINGSNUMMER } from '../util/constants';
+
+const behandlingsnummer = BEHANDLINGSNUMMER;
```

Comment:
Kan vi bruke BEHANDLINGSNUMMER direkte i staden for å mellomlagre i ein const? Evt definere behandlingsnummeret i denne fila i staden for i constants?